### PR TITLE
Added Environment Files

### LINKED ISSUE #70: Add `conda` environment configurations for Linux w/ CUDA and macOS w/ Metal
**Is your feature request related to a problem? Please describe.**
Because `tensorflow` is pinned to `2.8.0`, it is unresolvable for Apple Silicon macs. The `tensorflow` package for macOS on PyPI is under the name `tensorflow-macos` for `2.12` and earlier. If it's possible to unpin `tensorflow` and use anything later than 2.13 , then there isn't a need for `tensorflow-macos` , and pip install `tensorflow` would work.
For example when I run `pip install -e . `, I get the following error

```shell
ERROR: Could not find a version that satisfies the requirement tensorflow==2.8 (from cell-classification) (from versions: 2.13.0rc0, 2.13.0rc1, 2.13.0rc2, 2.13.0, 2.14.0rc0, 2.14.0rc1)
ERROR: No matching distribution found for tensorflow==2.8
```

**Describe the solution you'd like**
We should include various `conda` environment configuration files, one for CUDA on Linux / Windows, and one for MPS/metal (macOS gpu compute).

**Describe alternatives you've considered**
Stop using google products.

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by srivarra**
- added environment.yml files for macOS and Linux
- removed notebook output

**If you haven't already, please read through our [contributing guidelines](https://ark-analysis.readthedocs.io/en/latest/_rtd/contributing.html) before opening your PR**

**What is the purpose of this PR?**

Added an environment.yml file for Linux with CUDA support, and a macOS environment with Metal support.

**How did you implement your changes**

Bumped `tensorflow` to `2.13`.
Removed the function `normalize_data_format`.

**Remaining issues**

- [x] macOS environment file successfully creates the environment
- [ ] Linux environment file successfully creates the environment on Linux
- [ ] Linux environment file successfully creates the environment on WSL2
- [ ] macOS Tests pass
- [ ] Linux Tests pass

### CODE REVIEW

**Comment by JLrumberger** [comment]
Hi @srivarra,
what's the status on this, did you uncover some major roadblocks?

**Comment by srivarra** [comment]
@JLrumberger Yeah there are many issues with more recent versions of TensorFlow and Keras having incompatibilities. For example, some functions in Keras 2 (included in TF 2) are not available in newer TF versions, nor are they available in Keras 3.
